python/sparkdl/transformers/keras_transformer.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from sparkdl.param import keyword_only, HasKerasModel, HasInputCol, HasOutputCol
+
+from pyspark.ml import Transformer
+
+from sparkdl.transformers.keras_utils import KSessionWrap
+from sparkdl.graph.input import TFInputGraph
+from .tf_tensor import TFTransformer
+
+
+class KerasTransformer(Transformer, HasInputCol, HasOutputCol, HasKerasModel):
```

Comment:
Q: Where should I add a usage example? In the README? Or in the docstring potentially?

Reply:
I think the README is better for readability -- we'll create a real guide on the github page you made from the README.